### PR TITLE
server: add newline to the SSH key content

### PR DESCRIPTION
**Pull request by rhtyd**
On first startup, the management server instantiates and saves random
ssh keys in the database by calling a `trim()` on the string data. On
latest CentOS7 release (CentOS Linux release 7.6.1810 (Core)) I could
reproduce the issue where router_proxy (ssh) failed and kept
prompting for passphrase. By adding a newline to the id_rsa.cloud private
key it did not prompt the same. This may likely be an environmental bug.

The patch fixes the issue by adding a newline at the end of the generated
SSH public/private keys in the DB.

SSH packages details:
libssh2.x86_64                             1.4.3-12.el7_6.2            @updates
openssh.x86_64                             7.4p1-16.el7                @anaconda
openssh-askpass.x86_64                     7.4p1-16.el7                @base
openssh-clients.x86_64                     7.4p1-16.el7                @anaconda
openssh-server.x86_64                      7.4p1-16.el7                @anaconda

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan test

**Comment by rhtyd** [comment]
@blueorangutan package

File: plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtStartCommandWrapper.java [hunk 1]
```diff
@@ -116,7 +116,7 @@ public Answer execute(final StartCommand command, final LibvirtComputingResource
                     }
                 }
                 // try to patch and SSH into the systemvm for up to 5 minutes
-                for (int count = 0; count < 10; count++) {
+                for (int count = 0; count < 30; count++) {
```
Comment (DaanHoogland) [inline plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtStartCommandWrapper.java]: should the comment above be updated to read "up to 15 minutes"?

Comment (ustcweizhou) [reply plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtStartCommandWrapper.java]: @rhtyd is this change part of this commit ?

**Comment by rhtyd** [comment]
I'll re-test using a fresh centos7 based env. I hit the problem again today, this time with virtual routers but not ssvm/cpvm with this PR. It's probably something to do openssh-client changes or possibly my env.

**Comment by rhtyd** [comment]
I've developed a better fix, will send a new PR.

**Comment by rhtyd** [comment]
@ustcweizhou opened a new PR with more explaination here: https://github.com/apache/cloudstack/pull/3333

I also included a fix to ensure that ssh works on VR in case patching takes more time than anticipated.

**Pull request closed without merging**